library/db/src/main/kotlin/org/keynote/godtools/android/db/GodToolsDatabase.kt

```diff
@@ -77,6 +79,10 @@ class GodToolsDatabase @Inject internal constructor(@ApplicationContext private
                     }
                     43 -> db.execSQL(ToolTable.SQL_V43_ALTER_CATEGORY)
                     44 -> db.execSQL(TrainingTipTable.SQL_V44_CREATE_TRAINING_TIPS)
+                    45 -> {
```

Comment:
I think I understand what's happening here, but could use an explanation just to make sure 🙂

Reply:
Database migration to update the db definition for users that have previously installed the app.

Reply:
Okay, I was right 🙂 I was going to use the words "database migration", but didn't want to be completely wrong lol. I remember doing something similar in the missionhub app, but it was using redux instead of a database.